.circleci/config.yml

```diff
@@ -394,94 +394,94 @@ jobs:
           command: |
             (git submodule sync && git submodule update --init --jobs 8) & pip3 install -U lbuild & wait
       - run:
-          name: Full run of docs.modm.io-generator-script with upload
+          name: Full run of docs.modm.io-generator-script
           command: |
             export TERM=xterm-256color
             export COLUMNS=120
             python3 tools/scripts/docs_modm_io_generator.py -c -j4
       - run:
           name: Upload api documentation to docs.modm.io
+          when: always
```

Comment:
Don't think this is a good idea, since it may result in docs disappearing without warning.

Reply:
There is a warning: The CI job will be marked as "failed".
I think it's better if at least most of the documentation stays current instead of not updating it at all.

Reply:
I mean the CI failing on develop is already a big issue, we shouldn't add more issues by changing the website into some half-failed state.

Reply:
I've removed the merge commit from develop. The develop CI must not fail.

Reply:
WTF?!?

Reply:
> The develop CI must not fail.

Why?

Reply:
Have you really force-pushed to the develop branch?

Reply:
Cos the CI develop status is plastered all over the Readme and the homepage!
![Screenshot 2020-02-28 at 15 50 29](https://user-images.githubusercontent.com/163066/75558410-167f3c00-5a42-11ea-9b4d-245c50e2cf8b.png)
![Screenshot 2020-02-28 at 15 48 49](https://user-images.githubusercontent.com/163066/75558392-0f582e00-5a42-11ea-9c3a-e0dc24c4d454.png)

Reply:
> Have you really force-pushed to the develop branch

I should have done it right after the first failure.

Reply:
You should NEVER force-push to develop.

Reply:
> Cos the CI develop status is plastered all over the Readme and the homepage!

Oh no, two small red boxes. Horrible!!1! :laughing: :laughing: :laughing:

Reply:
Oh no, the maintainer making controversial decisions, cos you and I have different opinions about how we both messed up this PR. It's best if you stop contributing now entirely as payback and throw away all the work in your completely amazing PR that you worked on for months!

Let's fix this shit and move on.

Reply:
Srsly your work is amazing. Just don't make the CI fail ;-P

Reply:
I suggest to fix the CI using this commit: 5019700c0064770d67b5e8019359de0ff83a7dd1

Reply:
If the CI is not allowed to fail it is useless...

Reply:
What happened to those two commits?
![grafik](https://user-images.githubusercontent.com/2820734/75563047-ed62a980-5a49-11ea-9875-340f5d9109a3.png)